app/src/main/res/values/dimens.xml

```diff
@@ -1,10 +1,25 @@
 <resources>
-    <!-- Default screen margins, per the Android Design guidelines. -->
-    <dimen name="activity_horizontal_margin">16dp</dimen>
-    <dimen name="activity_vertical_margin">16dp</dimen>
-    <dimen name="fab_margin">16dp</dimen>
-    <dimen name="app_bar_height">200dp</dimen>
-    <dimen name="item_width">200dp</dimen>
-    <dimen name="text_margin">16dp</dimen>
-    <dimen name="spacing_medium">16dp</dimen>
+
+    <dimen name="margin_extra_small">4dp</dimen>
```

Comment:
@rohanagarwal94 1 padding across all spacings :rofl: Come on! Furthermore, who said that google development standards are constant

Also, the margin you marked are default android activity margins ( Not supposed to touch them)

Besides you gave an excellect example of how to use spacing. Let's create spacing_[micro, normal, large]